app/src/main/java/utils/ScheduleEvent.java

```diff
@@ -1,17 +1,21 @@
 package utils;
 
+import android.graphics.Color;
+
 import java.time.LocalTime;
 import java.util.ArrayList;
+import java.util.Calendar;
 
 public class ScheduleEvent {
 
     Activity activityType;
     String name;
-    LocalTime startTime;
-    LocalTime endTime;
-    ArrayList<Day> days;
+    Calendar startTime;
+    Calendar endTime;
```

Comment:
gets this from time selector, so will probably be easy to save/display in this format.